### PR TITLE
Fix components/TestApp.js, components/TestAutoSearchInput.js and components/TestPage.js

### LINKED ISSUE #11658: Update /components/TestAutoSearchInput.js to work with future dependencies
Update /components/TestAutoSearchInput.js to work with future dependencies

### LINKED ISSUE #11668: Update /components/TestPage.js to work with future dependencies
Update /components/TestPage.js to work with future dependencies

### PR DESCRIPTION
**Pull request by bobsilverberg**
~~Depends on https://github.com/mozilla/addons-frontend/pull/11723~~
Fixes #11657 
Fixes #11658 
Fixes #11668

### CODE REVIEW

**Comment by bobsilverberg** [comment]
As this depends on https://github.com/mozilla/addons-frontend/pull/11723, it probably makes sense to hold off reviewing it until that lands, and then I will rebase this.

**Review by eviljeff — APPROVED** [review]

**Pull request merged**
